static/styles/zulip.scss

```diff
@@ -565,7 +563,7 @@ li.actual-dropdown-menu i {
 
 .message_area_padder {
     /* The height of the header and the tabbar plus a small gap */
-    margin-top: 93px;
+    margin-top: 57px;
```

Comment:
For testing this commit, a thing to verify is that the loading indicator still correctly circles the top-messages-logo.

`manage.py populate_db -n2000` might be helpful in creating a large enough message history locally that you can trigger fetches at the top of the your scroll history.